Description: Remove google custom engine queries script
 Remove javascript for the custom google engine queries, in the header
 of each html documentation page, to avoid potential privacy breach.
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Origin: other
Forwarded: no
Last-Update: 2016-09-29

--- a/doc/html/header.txt
+++ b/doc/html/header.txt
@@ -12,18 +12,6 @@
 	<link rel="stylesheet" type="text/css" href="reset.css" />
 	<link rel="stylesheet" type="text/css" href="style.css" />
 	<link rel="stylesheet" type="text/css" href="slurm.css" />
-
-	<script type="text/javascript">
-	(function() {
-	  var cx = '011890816164765777536:jvrtxrd3f0w';
-	  var gcse = document.createElement('script');
-	  gcse.type = 'text/javascript';
-	  gcse.async = true;
-	  gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
-	  var s = document.getElementsByTagName('script')[0];
-	  s.parentNode.insertBefore(gcse, s);
-	})();
-	</script>
 </head>
 
 <body>
